### PR TITLE
Removed Javadoc warnings

### LINKED ISSUE #2808: Remove Javadoc warnings
There are lots...roll up your sleeves.

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #2808

Also removed a few warnings from asciidoc generation

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc [hunk 1]
```diff
@@ -139,48 +139,6 @@ You can use the `--random-route` option to avoid this when you push the server a
 NOTE: If you need to configure multiple Maven repositories, a proxy, or authorization for a private repository, see link:http://docs.spring.io/spring-cloud-dataflow/docs/{scdf-core-version}/reference/htmlsingle/#getting-started-maven-configuration[Maven Configuration].
 
 
-[[getting-started-cloudfoundry-deploying-using-manifest]]
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc]: Is there a reason to delete this?

File: spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/service/AppRegistryService.java [hunk 2]
```diff
@@ -72,6 +72,7 @@
 
 	/**
 	 * Deletes all provided {@link AppRegistration}'s.
+	 * @param appRegistrations the {@link AppRegistration}s that should be deleted
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/service/AppRegistryService.java]: We can avoid saying `should` as these app registrations might not get deleted if they are in use.

File: spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc (hunk 1 above)
Comment (cppwfs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-cloudfoundry.adoc]: Hey Ilaya!    Its a duplicate section of what is found at line 244

File: spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/service/AppRegistryService.java (hunk 2 above)
Comment (cppwfs) [reply spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/service/AppRegistryService.java]: Thanks.  Will update.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Rebased and merged as 782a421d

**Pull request closed without merging**